mappings/net/minecraft/world/gen/chunk/ChunkGenerator.mapping

```diff
@@ -7,44 +7,69 @@ CLASS net/minecraft/unmapped/C_kgfalbim net/minecraft/world/gen/chunk/ChunkGener
 		COMMENT <p>
 		COMMENT This is used by {@link FlatChunkGenerator} to overwrite biome properties like whether lakes generate,
 		COMMENT while preserving the original biome identifier.
+	FIELD f_tgxfgajp generationSettingsGetter Ljava/util/function/Function;
```

Comment:
Could be shortened to `settingsGetter`

Reply:
I like that idea; will commit it later when I get to my main computer